primitives/npos-elections/src/tests.rs

```diff
@@ -792,6 +792,13 @@ mod assignment_convert_normalize {
 
 mod score {
 	use super::*;
+	use crate::ElectionScore;
+	use sp_arithmetic::PerThing;
+
+	fn is_score_better(this: [u128; 3], that: [u128; 3], p: impl PerThing) -> bool {
+		ElectionScore::from(this).strict_threshold_better(ElectionScore::from(that), p)
```

Comment:
imho we should remove `is_score_better` and just use `.strict_threshold_better` with explicit `ElectionScore` structs. Otherwise it semi defeats the purpose of creating the new struct for readability